app/views/imports/index.html.haml

```diff
@@ -15,7 +15,7 @@
       - for import_result in import_results
         %tr
           %td
-            %a{ href: import_result.uri }= import_result.name
+            %a{ href: import_path(import_result.id) }= import_result.name
```

Comment:
I meant to mention this earlier, but forgot - I think this could just as easily be:

```
%td= link_to import_result.name, import_path(import_result.id)
```